src/style.css

```
@@ -0,0 +1,97 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+}
+
+.column {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  justify-content: space-evenly;
+  align-content: stretch;
+  order: 0;
+  flex-grow: 1;
+  flex-shrink: 0;
+  min-width: 300px;
+  height: 300px;
+  color: white;
+  font-size: 100px;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+@media  (min-width: 450px) and (max-width: 599px) {
+  .container {
+    flex-direction: row;
+  }
+}
+
+@media (min-width: 600px) and (max-width: 899px) {
```

```suggestion
@media (min-width: 600px) {
```
Нижний медиа запрос перекроет этот на ширине 900 пикселей